Cargo.lock

```diff
@@ -3065,6 +3065,7 @@ dependencies = [
  "migration-core",
  "mongodb",
  "mongodb-client",
+ "parking_lot 0.12.1",
```

Comment:
I think it would be worth digging into why we have multiple versions of parking lot in Cargo.lock

Reply:
metrics-exporter-prometheus uses 0.11...